oaknational/Oak-Web-Application
# [ENG-79] Feat/isr error handling

## Description

- adds `getPageProps()` wrapper function

## How to test

1. Go to[ {cloud link}](https://deploy-preview-1727--oak-web-application.netlify.thenational.academy/beta/teachers-2023/programmes/physics-secondary-ks4/units/measuring-and-calculating-motion/lessons)
2. You should see the cached version of the page
3. You should see errors coming through to bugsnag like: https://app.bugsnag.com/oak-national-academy/oak-web-application/timeline?filters[event.since]=2023-06-27T11%3A48%3A48.812Z&filters[error.status]=open&filters[event.before]=2023-06-27T14%3A40%3A00.000Z&sort=last_seen

<img width="958" alt="image" src="https://github.com/oaknational/Oak-Web-Application/assets/12934669/cbde74ec-64bd-45fd-9442-b929b78913d4">
